datasets/brk/aantekeningenkadastraleobjecten/v1.1.0.json

```diff
@@ -82,14 +82,14 @@
         "description": "Identificatie van het betrokken subject"
       },
       "heeftBetrekkingOpKadastraalObject": {
-        "shortname": "heeftBetrekkingOpKadast",
+        "shortname": "HftBtrkOpKot",
```

Comment:
mag shortname beginnen met een hoofdletter?

Reply:
hm meta-schema staat elke string toe voor shortname. Dat moeten we misschien wat strakker zetten

Reply:
Waarom zou je dat willen, die hoofdletter? Het gaat er hier vooral om, om een naam te bieden voor de tabel. Dus het wordt sowieso gesnakecased. In de schemas gebruiken we overal camelcase, waarom zouden we dan hier pascalcase voor gebruiken?